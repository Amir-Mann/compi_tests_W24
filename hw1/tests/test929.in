JE->S%E?'L8k5]pahZK{0\F:[Fr/y;;]C7]_za'Bwj#TLkupU@O}?gWFoB"l%nTM_x3.\*h-0B/N7W|_bF