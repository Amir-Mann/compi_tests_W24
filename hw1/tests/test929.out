1 ID JE
Error 
